modules/search/reddit.py

```diff
@@ -20,59 +20,83 @@
 meta = {
 	'name': 'Reddit Search',
 	'author': 'Kunal Khandelwal',
-	'version': '0.5',
+	'version': '1.0',
 	'description': 'Search your query in the Reddit and show the results.',
-	'sources': ('google', 'yahoo', 'bing', 'duckduckgo', 'metacrawler', 'millionshort', 'carrot2', 'qwant'),
+	'sources': ('google', 'yahoo', 'bing', 'yippy', 'metacrawler', 'millionshort', 'carrot2', 'qwant', 'reddit_pushshift'),
```

Comment:
your changes have a conflict with the new changes. I mean removing yippy.

Reply:
Done